tools/wasm2c/__init__.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 The Emscripten Authors.  All rights reserved.
```

Comment:
I don't care much about the location of the new file, but a benefit to `wasm2c/__init__.py` is that it won't mess up autocomplete (which having both `wasm2c.py` and `wasm2c/*.c` would do).

Reply:
It think just having tools/wasm2c.py is fine.

I don't think we should worry too much about auto-complete when naming files.

Reply:
Ok, renamed as you requested.